src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -21,19 +22,30 @@
 
 namespace Microsoft.DotNet.DarcLib
 {
-    public class AzureDevOpsClient : IGitRepo
+    public class AzureDevOpsClient : RemoteRepoBase, IGitRepo
     {
         private const string DefaultApiVersion = "5.0-preview.1";
 
-        private static readonly Regex repoUriPattern = new Regex(@"^/dnceng/(?<team>[^/]+)/_git/(?<repo>[^/]+)$");
+        private static readonly string CommentMarker =
+            "\n\n[//]: # (This identifies this comment as a Maestro++ comment)\n";
 
-        private static readonly Regex prUriPattern = new Regex(
-            @"^/(?<team>[^/]+)/_apis/git/repositories/(?<repo>[^/])/pullRequests/(?<id>\d+)$");
+        private static readonly Regex repositoryUriPattern = new Regex(
```

Comment:
>repositoryUriPattern [](start = 38, length = 20)

The naming style of this project is... unclear.  Looking at some other classes, it looks like in general "static readonly" things are treated like consts, and should be "RepositoryUriPattern"